etl/scripts-ccao-data-warehouse-us-east-1/ccao/ccao-condominium_parking.R

```diff
@@ -65,6 +65,7 @@ nonlivable[["neg_pred"]] <- map(
 # Upload all nonlivable spaces to nonlivable table
 nonlivable %>%
   bind_rows() %>%
+  mutate(loaded_at = as.character(Sys.time())) %>%
```

Comment:
**question (blocking):** We want all the `loaded_at` columns across our data to share the same format, precision, and type. Does this yield the same type as the `DATE_FORMAT` calls in sql?

Reply:
`select loaded_at from ccao.pin_nonlivable limit 1` 2024-12-17 16:15:22.613758

`select loaded_at from iasworld.pardat limit 1` 2024-12-21 07:46:36.530

There's slightly more precision for the columns generated by r, is that a concern? It seemed a little weird to remove precision in order to make string lengths match when they're still comparable:

`select '2024-12-21 07:46:36.530' < '2024-12-17 16:15:22.613758'` false
`select '2024-12-21 07:46:36.530' > '2024-12-17 16:15:22.613758'` true

Reply:
Nope, that's totally fine. Thanks for checking.